### ISSUE TITLE
Add Traikālyaparīkṣā to EAST? (Not in SUEBS)

### ISSUE THREAD

**Issue by paddymcall** [ea-east/east-biblio#17]
Concerning Peking 5705: Dignāga's Traikālyaparīkṣā is not mentioned in
SUEBS, probably because it is not a pramāṇa work and largely just a
remodeling of several verses from Bhartṛhari.  In the introduction to
SUEBS, p. XVI, Steinkellner and Much argue for their inclusion of two
of Dignāga’s early works, the \*Upādāyaprajñaptiprakaraṇa and the
Ālambanaparīkṣā, on the grounds that these had a strong influence on
the Pramāṇasamuccaya.  Following this strategy, we should expect all
of Dignāga’s early and not-quite-pramāṇa works to be excluded from
SUEBS. However, given that recent research on the Pramāṇasamuccaya has
shown how strong an influence Bhartṛhari had on the ideas developed
there, I’d suggest to include this work in EAST too, with a note
cautioning that, acc. to Frauwallner (1959,
https://east.ikga.oeaw.ac.at/bib/5484/, p. 115), it contains two ideas
that diverge from his later positions: first, Dignāga there holds that
cognition is unchanging (“ewig und unveränderlich”, /rnam par mi ’gyur
ba/); second, Dignāga does not differentiate there between perception
and conceptual cognition.

1) [x] Add to EAST for Dignāga
2) [ ] Add to bibliography
3) [ ] Add http://www.jits-ryukoku.net/data/20/ick20_nasu.pdf, in the correct relation (@pchiou6932)

As you, Patrick, pointed out already, {Peking} 5705 is missing. The following bibliographical data should be added:
dus gsum brtag pa {co ne ce 87a6–88b2; {Peking} 5705 ce 179a4–180b1; sde dge 4207 ce 87b2-88b5; snar thang ce 182a3-183b1}

_Originally posted by @JamieCreek in https://github.com/ea-east/east-biblio/issues/5#issuecomment-610438631_

**Comment by pchiou6932** [comment]
Following this line, I would like to come back to an issue we discussed before, namely, to include works of the thinkers of the Pramāṇa tradition that are not pramāṇa-oriented but contain considerable discussions related to pramāṇa, ex. Kamalaśīla's Madhyamakāloka. Maybe we can create a new category for this kind of works, labeling them as ''pramāṇa-related treatises''?

**Comment by kellner** [comment]
I agree that the Traikālyaparīkṣā should be included. For other works, criteria are needed. I would propose that a list of such works that are currently not included, but should be included, is produced, and that this list is discussed at our next meeting.

**Comment by kellner** [comment]
We are still waiting on a list for this, I guess.
